yb-voyager/cmd/exportData.go

```diff
@@ -388,7 +388,14 @@ func exportData() bool {
 				utils.ErrExit("error checking if replication slot is active: %v", err)
 			}
 			if isActive {
-				utils.ErrExit("Replication slot '%s' is active. Check and terminate if there is any internal voyager process running.", msr.PGPublicationName)
+				pid, err := dbzm.GetPIDOfDebeziumOnExportDir(exportDir, exporterRole)
```

Comment:
slight restructure here: 

The core error here is that the replication is already active, which is something we want to report. I think it makes sense to  add the PID information on a best-effort basis (it is possible that the lock file is not there because dbzm is not running; it is possible we run into some other weird error with the lock file?, etc). 

So can we structure it like this: 
- form the base error message. 
- check for pid in lock file. If we get the pid, add the pid to the error message. 
- throw the error.

Does that make sense?

Reply:
yeah make sense! made the change